README.md

```diff
@@ -0,0 +1,163 @@
+# selenium-sandbox
+
+easily mock a web application accessed through selenium. Contains also an environment for integrating with jest.
+
+## Getting started
+
+### 0. Install
+
+    yarn add -D @4c/selenium-sandbox
+
+### 1. Add the sandbox to your test client
+
+First, we need to instrument our application to use the sandbox environment. Create a module (let's call it `injectSandbox.js`)
+
+```js
+import sandbox from '@4c/selenium-sandbox/lib/sandbox';
+
+const store = {
+  widgets: [],
+};
+
+// additional properties and methods that can be accessed from selenium. for example here we are passing a store object
+const context = { store };
+
+sandbox.setupTextContext(context);
+
+sandbox.fetchMock.mock('path:/api/v1/widgets', () => store.widgets);
+```
+
+### 2. Instrument Selenium
+
+Add utilities to an already existing selenium driver (v4):
+
+```js
+import { augmentDriver } from '@4c/selenium-sandbox/lib/selenium';
+
+const driver = augmentDriver(
```

Comment:
Why "augment" here and build below vs an overload on the same method

Reply:
`augmentDriver` extends an existing driver, while `buildDriver`, well, builds one from scratch. does it make sense?